app/domain/authentication/authn_jwt/configuration_interface.rb

```diff
@@ -2,6 +2,7 @@ module Authentication
   module AuthnJwt
     # Interface containing JWT configuration functions to be implemented in JWTConfiguration class for a vendor
     class ConfigurationInterface
+      def create_authentication_parameters(authentication_parameters); end
```

Comment:
this function get as input `authenticator_input`

Reply:
Not in this pr :-)